security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+[[security_and_privacy_of_ln]]
+== Security and Privacy of the Lightning Network
+In this chapter, we look at the issues related to the security and privacy of the LN.
+
+= Why is privacy important? =
+
+The key value proposition of cryptocurrency is censorship resistant money.
+It has been shown many times that the power of governments over money is often abused.
+Bitcoin offers individuals the possibility of storing and transferring their wealth without restrictions.
+
+Lightning continues this mission.
+It aims at providing a better user experience with instant payments while not sacrificing censorship resistance.
+However, the LN operates under a different security model, which introduces novel security and privacy challenges.
+
+= Definitions of privacy =
+
+Privacy is a complex topic.
+It is hard to precisely define what we mean by privacy.
+Therefore, the question: "Is Lightning private?" has no direct answer.
+Privacy researchers approach this issue as follows.
+First, they define a _security model_ that specifies what an adversary is capable of and aims to achieve.
+Then, they describe the relevant properties of the system and check whether it conforms to the requirements.
+A security model is based on a set of underlying _security assumptions_.
+In cryptographic systems, they are often centered around the mathematical properties of the cryptographic primitives such as ciphers, signatures, and hash functions.
+The security assumptions of the Lightning Network are that the ECDSA signatures, SHA-256 hash function, and other cryptographic functions used in the protocol behave within their security definitions.
+For example, we assume that it is practically impossible to find a pre-image of a hash function.
+This allows the LN to rely on the HTLC mechanism for the atomicity of multi-hop payments: nobody except the final recipient can reveal the payment secret and resolve the HTLC.
+We also assume a degree of connectivity in the network, namely, that Lightning channels form a connected graph (therefore, it is possible to find a path from any sender to any receiver) and that network messages get propagated within certain timeouts.
+
+Here are some possible models of adversaries in the Lightning Network.
+A regular "honest-but-curious" forwarding node can observe payment amounts, the immediately preceding and following nodes, the graph of announced channels with their capacities.
```

Comment:
nit: "payment amounts" is a bit misleading here IMO, because there's a difference between the amount paid and the amount forwarded through a specific hop along the route.